guides/customization/tailscale.md

```diff
@@ -133,29 +128,30 @@ Pin: origin pkgs.tailscale.com
 Pin-Priority: 500
 ```
 
-## Step 1c: Add signing key for Tailscale package repository
+### Step 1c: Add the signing key for Tailscale package repository
 
-Retrieve the signing key from Tailscale and store the binary (dearmored) key
+Retrieve the signing key from Tailscale, and store the binary (dearmored) key
 file in `files/usr/share/keyrings/tailscale.gpg`:
 
 ```console
 $ curl --silent --show-error --location "https://pkgs.tailscale.com/stable/ubuntu/focal.gpg" | \
     gpg --dearmor --yes --output=files/usr/share/keyrings/tailscale.gpg
 ```
 
-## Step 1d: Override default tailscaled service settings
+### Step 1d: Override default `tailscaled` service settings
 
-By default, tailscaled will store its internal state in a "state file" located
-at `/var/lib/tailscale/tailscaled.state`, which is ephemeral in Coder. We will
-need to modify the service settings in order to:
+By default, `tailscaled` will store its internal state in a **state file**
+located at `/var/lib/tailscale/tailscaled.state` (this is is ephemeral in
+Coder). We will need to modify the service settings to:
 
 - Store the state file in the persistent home volume (`/home/coder`)
 - Enable userspace networking
 - Enable the SOCKS5 proxy (optional)
 - Enable the HTTP proxy (optional)
 
 If you do not require outbound connections from the workspace to other services
-running in the tailnet, you may skip configuring the proxies.
+running in the tailnet, you may skip the final two steps configuring the
```

Comment:
we'll have to be careful if we add steps to also modify this :thinking: is there a good way to phrase this that doesn't refer to the position?